src/IO/InputStreamFromROOTFile.cxx

```diff
@@ -86,7 +89,9 @@ InputStreamFromROOTFile::initialise_keymap()
     this->parser.add_key("name of data file", &this->filename);
     this->parser.add_key("Singles readout depth", &this->singles_readout_depth);
     this->parser.add_key("name of input TChain", &this->chain_name);
+    this->parser.add_key("exclude true events", &this->exclude_trues);
     this->parser.add_key("exclude scattered events", &this->exclude_scattered);
+    this->parser.add_key("exclude unscattered events", &this->exclude_unscattered);
     this->parser.add_key("exclude random events", &this->exclude_randoms);
```

Comment:
Are we happy with these keys, `exclude true events` in particular? This refers to any event from the SAME eventID

Reply:
I'm sure you're not going to like `exclude non-random events`, although then we can have a triple negative!

what about `exclude events with same ID`

Reply:
`exclude non-random events` is cleaner and it is obvious it the the oposite of `exclude random events`... 😞 I'll do it